File: app/src/main/java/com/dpis/module/home/ModeHelpActivity.java
package com.dpis.module.home;

import com.dpis.module.settings.LocalizedActivity;
import com.dpis.module.about.presentation.SupportActivityContent;

import android.os.Bundle;

public final class ModeHelpActivity extends LocalizedActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        SupportActivityContent.installModeHelp(this);
    }
}

File: app/src/main/java/com/dpis/module/home/ModeHelpActivity.kt
package com.dpis.module.home

import android.os.Bundle
import com.dpis.module.home.presentation.installModeHelp
import com.dpis.module.settings.LocalizedActivity

class ModeHelpActivity : LocalizedActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        installModeHelp()
    }
}
